[2024-01-17T19:03:45.308-0300] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cities_coleta_teste.df scheduled__2024-01-17T22:02:32.598644+00:00 [queued]>
[2024-01-17T19:03:45.311-0300] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cities_coleta_teste.df scheduled__2024-01-17T22:02:32.598644+00:00 [queued]>
[2024-01-17T19:03:45.311-0300] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-01-17T19:03:45.332-0300] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): df> on 2024-01-17 22:02:32.598644+00:00
[2024-01-17T19:03:45.335-0300] {standard_task_runner.py:57} INFO - Started process 81906 to run task
[2024-01-17T19:03:45.337-0300] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'cities_coleta_teste', 'df', 'scheduled__2024-01-17T22:02:32.598644+00:00', '--job-id', '2429', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpy4glhalo']
[2024-01-17T19:03:45.338-0300] {standard_task_runner.py:85} INFO - Job 2429: Subtask df
[2024-01-17T19:03:45.364-0300] {task_command.py:416} INFO - Running <TaskInstance: cities_coleta_teste.df scheduled__2024-01-17T22:02:32.598644+00:00 [running]> on host desktop
[2024-01-17T19:03:45.427-0300] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='datasetpibic@gmail.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='cities_coleta_teste' AIRFLOW_CTX_TASK_ID='df' AIRFLOW_CTX_EXECUTION_DATE='2024-01-17T22:02:32.598644+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-17T22:02:32.598644+00:00'
[2024-01-17T19:03:51.409-0300] {logging_mixin.py:154} INFO - Called | 1705529031.4070334 | df
[2024-01-17T19:03:51.416-0300] {logging_mixin.py:154} INFO - SAVING FILE df
[2024-01-17T19:03:51.417-0300] {python.py:194} INFO - Done. Returned value was: None
[2024-01-17T19:03:51.419-0300] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=cities_coleta_teste, task_id=df, execution_date=20240117T220232, start_date=20240117T220345, end_date=20240117T220351
[2024-01-17T19:03:51.498-0300] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-01-17T19:03:51.505-0300] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
